apps/console/src/features/identity-providers/components/identity-provider-list.tsx

```diff
@@ -440,9 +442,12 @@ export const IdentityProviderList: FunctionComponent<IdentityProviderListPropsIn
                         primaryAction={ t("common:confirm") }
                         secondaryAction={ t("common:cancel") }
                         onSecondaryActionClick={ (): void => setShowDeleteConfirmationModal(false) }
-                        onPrimaryActionClick={
-                            (): void => handleIdentityProviderDelete(deletingIDP.id)
-                        }
+                        onPrimaryActionClick={ (): void => {
+                            setLoading(true);
+                            handleIdentityProviderDelete(deletingIDP.id).finally(() => {
```

Comment:
We can abstract this function too as in the above comment

Reply:
Addressed via af36e121eed12e6c0f06003fee7e01072b7c4ee6